src/lib/php/Dao/ClearingDao.php

```diff
@@ -357,7 +357,7 @@ public function createDecisionFromEvents($uploadTreeId, $userId, $groupId, $decT
     $uploadTreeTable = $this->uploadDao->getUploadtreeTableName($uploadId);
     $itemTreeBounds = $this->uploadDao->getItemTreeBounds($uploadTreeId, $uploadTreeTable);
 
-    if ($this->isDecisionIrrelevant($uploadTreeId, $groupId)) {
+    if ($this->isDecisionCheck($uploadTreeId, $groupId, 'IRR')) {
```

Comment:
How about using class constants?

```suggestion
    if ($this->isDecisionCheck($uploadTreeId, $groupId, DecisionTypes::IRRELEVANT)) {
```